python/tests/sql/test_predicate.py

```diff
@@ -174,7 +174,7 @@ def test_st_equals_for_st_linestring_and_st_polygon(self):
 
         string = "100.01,200.01,100.5,200.01,100.5,200.5,100.01,200.5,100.01,200.01"
         equal_df = self.spark.sql(
-            f"select * from polygonDf where ST_Equals(polygonDf.polygonshape, ST_LineStringFromText('{string}', ',')) "
+            f"select * from polygonDf where ST_Equals(polygonDf.polygonshape, ST_LineStringFromText('{string}', ',')) "  # nosec B608
```

Comment:
Can we skip checking source files in python/tests directory? Tests are not exposed as attack surface so I believe that it is reasonable to skip checking test sources.